scripts/build/Jenkins/Jenkinsfile

```diff
@@ -200,6 +200,14 @@ def GetEnvStringList(Map envVarMap) {
     return strList
 }
 
+def GetCrashArtifactDir() {
+    if (env.IS_UNIX) {
+        return '/var/lib/apport/coredump/'
+    } else {
+        return "$LOCALAPPDATA\\CrashDumps"
```

Comment:
This may need another condition for Mac, as that has its own crash dump location (`~/Library/Logs/DiagnosticReports/`), but that depends on whether we want dumps on that platform.

Reply:
Example code to get Mac vs Linux

```
if (env.IS_UNIX) {
    def uname = sh(script: 'uname', returnStdout: true).trim()
    if (uname.startsWith('Darwin')) {
        return '~/Library/Logs/DiagnosticReports/' // MacOS
    }
    else {
        return '/var/lib/apport/coredump/' // Linux
    }
}
```

Reply:
added Mac